### ISSUE TITLE
Two crashes in the v1.0.7

### ISSUE THREAD

**Issue by florianmlr** [crisp-im/crisp-sdk-android#79]
Hello, on crashlytics I noticed 2 crashes that occur multiple times : 
```
Caused by android.content.res.Resources$NotFoundException
Unable to find resource ID #0x7f0800ba
android.content.res.ResourcesImpl.getResourceName (ResourcesImpl.java:253)
android.view.LayoutInflater.inflate (LayoutInflater.java:423)
im.crisp.client.b.e.b.a.onCreateView (a.java:3)
androidx.fragment.app.Fragment.performCreateView (Fragment.java:2699)
com.android.internal.os.ZygoteInit.main (ZygoteInit.java:807)
```
and
```
Fatal Exception: java.lang.NullPointerException
Attempt to invoke virtual method 'java.lang.Object android.content.Context.getSystemService(java.lang.String)' on a null object reference
im.crisp.client.b.d.d.b.g (b.java:6)
okhttp3.internal.http.RealInterceptorChain.request (RealInterceptorChain.java:117)
okhttp3.internal.http.RealInterceptorChain.proceed (RealInterceptorChain.java:147)
okhttp3.internal.connection.ConnectInterceptor.intercept (ConnectInterceptor.java:45)
okhttp3.internal.http.RealInterceptorChain.proceed (RealInterceptorChain.java:147)
okhttp3.internal.http.RealInterceptorChain.proceed (RealInterceptorChain.java:121)
okhttp3.internal.cache.CacheInterceptor.intercept (CacheInterceptor.java:93)
okhttp3.internal.http.RealInterceptorChain.proceed (RealInterceptorChain.java:147)
okhttp3.internal.http.RealInterceptorChain.proceed (RealInterceptorChain.java:121)
okhttp3.internal.http.BridgeInterceptor.intercept (BridgeInterceptor.java:93)
okhttp3.internal.http.RealInterceptorChain.proceed (RealInterceptorChain.java:147)
okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept (RetryAndFollowUpInterceptor.java:127)
okhttp3.internal.http.RealInterceptorChain.proceed (RealInterceptorChain.java:147)
okhttp3.internal.http.RealInterceptorChain.proceed (RealInterceptorChain.java:121)
okhttp3.RealCall.getResponseWithInterceptorChain (RealCall.java:257)
okhttp3.RealCall$AsyncCall.execute (RealCall.java:201)
okhttp3.internal.NamedRunnable.run (NamedRunnable.java:32)
```
(similar to this issue https://github.com/crisp-im/crisp-sdk-android/issues/76#issue-1011671538 )

Will there be a release soon?

**Comment by Doc1faux** [comment]
Hi @florianmlr and thank you for your feedback.
Sorry for my late reply, I was on the brand new Crisp Android app :)
The crash which is also reproduced in #76 has been fixed in the recently released 1.0.9.

The one about the `android.content.res.Resources$NotFoundException` exception is a bit strange...
I prefixed all res in the SDK with `crisp_` in the 1.0.7 release to avoid conflicts with overridden res in apps (see #59).
In 1.0.9, in order to try to fix this crash, I also prefixed all ids with `crisp_`, let me know if you encounter this particular crash again after updating the SDK.
